README.md

```diff
@@ -1,20 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# React-Redux setup
+# Covid Metrics
```

Comment:
- [ ] You seem to have raised a Pull Request from `develop` to `main`. But according to the [link for correct Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) provided in your project requirements, you should raise a Pull Request from the `feature` branch to the `develop` instead. Kindly fix this.